packages/datadog-plugin-graphql/test/index.spec.js

```diff
@@ -1535,91 +1614,92 @@ describe('Plugin', () => {
             })
         })
       })
-
-      withVersions('graphql', 'apollo-server-core', apolloVersion => {
-        let runQuery
-        let mergeSchemas
-        let makeExecutableSchema
-
-        before(() => {
-          tracer = require('../../dd-trace')
-
-          return agent.load('graphql')
-            .then(() => {
-              graphql = require(`../../../versions/graphql@${version}`).get()
-
-              const apolloCore = require(`../../../versions/apollo-server-core@${apolloVersion}`).get()
-              const graphqlTools = require(`../../../versions/graphql-tools@3.1.1`).get()
-
-              runQuery = apolloCore.runQuery
-              mergeSchemas = graphqlTools.mergeSchemas
-              makeExecutableSchema = graphqlTools.makeExecutableSchema
-            })
-        })
-
-        after(() => {
-          return agent.close({ ritmReset: false })
-        })
-
-        it('should support apollo-server schema stitching', done => {
-          agent
-            .use(traces => {
-              const spans = sort(traces[0])
-
-              expect(spans).to.have.length(3)
-
-              expect(spans[0]).to.have.property('name', expectedSchema.server.opName)
-              expect(spans[0]).to.have.property('resource', 'query MyQuery{hello}')
-              expect(spans[0].meta).to.not.have.property('graphql.source')
-
-              expect(spans[1]).to.have.property('name', 'graphql.resolve')
-              expect(spans[1]).to.have.property('resource', 'hello:String')
-
-              expect(spans[2]).to.have.property('name', 'graphql.validate')
-              expect(spans[2].meta).to.not.have.property('graphql.source')
-            })
-            .then(done)
-            .catch(done)
-
-          schema = mergeSchemas({
-            schemas: [
-              makeExecutableSchema({
-                typeDefs: `
-                  type Query {
-                    hello: String
-                  }
-                `,
-                resolvers: {
-                  Query: {
-                    hello: () => 'Hello world!'
-                  }
-                }
-              }),
-              makeExecutableSchema({
-                typeDefs: `
-                  type Query {
-                    world: String
-                  }
-                `,
-                resolvers: {
-                  Query: {
-                    world: () => 'Hello world!'
-                  }
-                }
-              })
-            ]
-          })
-
-          const params = {
-            schema,
-            query: 'query MyQuery { hello }',
-            operationName: 'MyQuery'
-          }
-
-          runQuery(params)
-            .catch(done)
-        })
-      })
+      // TODO: this is an outdated test, comeback to it to officially add support for
```

Comment:
Can you elaborate on this test removal more? At first glance it looks like this PR breaks support for `apollo-server-core`.

Reply:
Let's also call `.skip` on the test instead of commenting it out.

Reply:
@tlhunter 

The precense of graphql@^15.2.0 in the /versions folder causes graphql-tools@3.1.1
to break in the before() hook. This test tests a library that had it's last release occur 5 years ago &
updating the test would require using newer version of apollo-core which have a completely different syntax
and produces traces that are different then what is expected by this test